projects/js-packages/publicize-components/src/components/share-buttons/styles.module.scss

```diff
@@ -32,3 +32,14 @@ $whatsapp: #2ab318;
 		color: var(--jp-black);
 	}
 }
+
+.container {
+	display: flex;
+}
+
+.label.label {
+	font-size: 15px;
```

Comment:
Looking at the designs, the text should be 13px to match the title of the panel. It is looking a bit big for me.